### DISCUSSION TITLE
Code review IM-13

### DISCUSSION THREAD

**Discussion by Alzex** [HowProgrammingWorks/LiveQA#109] [Code review]
Zemlianskyi Oleksandr IM-13

This a simple version of classic 'Sea battle' game in web which can be played by 2 people offline. All basic functionality is done. It may only need some minor fixes and improvements.

Repository: https://github.com/Alzex/Sea-Battle

**Comment by tshemsedinov** [comment]
See: https://github.com/Alzex/Sea-Battle/issues/1
